### ISSUE TITLE
Nginx option

### ISSUE THREAD

**Issue by Park0** [inoxio/ansible-role-ispconfig3#5]
I would like to use ISP-config with nginx. I used the follow code:

```
---
- hosts: all
  become: true

  roles:
    - role: inoxio.ispconfig3
      tags: ispconfig
      mail_admin_email: isp-config@domain.lan
      mail_base_domain: domain.lan
      ispconfig_hostname: server.lan
      roundcube_hostname: mail.lan
      phpmyadmin_hostname: phpmyadmin.lan
      ispconfig_configure_firewall: n
      ispconfig_do_backup: n
      ispconfig_configure_apache: n
      ispconfig_configure_nginx: y

```
But i still get apache2.

**Comment by MartinLesser** [comment]
Hello Park0,

thank you for the issue. I looked into it. The problem seems to be that we developed the role with only apache in mind. The file [apache_php_configuration](https://github.com/inoxio/ansible-role-ispconfig3/blob/master/tasks/apache_php_configuration.yml) installs apache. 
The `ispconfig_configure_nginx` variable will be commited to the ISPConfig installation as an ini-file. So I think nginx should be installed too. I will ask the team if we will implement nginx support. But I can't promise anything because we are busy with other projects right now. 

Kind regards,
Martin
